lib/auth.ts

```diff
@@ -175,9 +175,27 @@ export default class BalenaAuth {
 		return token.isValid();
 	};
 
+	/**
+	 * @member get2FAStatus
+	 * @summary Gets whether passing a 2FA challenge is pending, passed or not required.
+	 * @function
+	 * @public
+	 *
+	 * @returns {Promise<'not_required'|'pending'|'passed'>}
+	 *
+	 * @example
+	 * auth.get2FAStatus().then((get2FAStatus) => { ... });
+	 */
+	public get2FAStatus = async (): Promise<
+		'not_required' | 'pending' | 'passed'
```

Comment:
I think that's way more useful & straight-forward than having separate `has2FA` and `needs2FA` methods.
I hope it will eliminate the "can you explain me how bC 2fa works in the jwt" question.